src/components/LandingPage.css

```
@@ -0,0 +1,193 @@
+/* ── Landing Page ───────────────────────────────────────────── */
+.landing {
+  min-height: 100dvh;
+  background: linear-gradient(160deg, #fdf8ee 0%, #f7edce 60%, #ede0b8 100%);
+  font-family: 'Inter', sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)